compiler/rustc_infer/src/infer/lexical_region_resolve/mod.rs

```diff
@@ -96,6 +97,7 @@ pub enum RegionResolutionError<'tcx> {
         Region<'tcx>,
         SubregionOrigin<'tcx>,
         Region<'tcx>,
+        Vec<Span>,
```

Comment:
needs a comment!

Reply:
are these the "spans" associated with "influencer constraints"?

Reply:
Yes.